embedded/embedded-test/src/test/java/org/mule/runtime/module/embedded/ApplicationTestCase.java

```diff
@@ -78,6 +80,11 @@
   @Rule
   public TemporaryFolder temporaryFolder = new TemporaryFolder();
 
+  @AfterClass
+  public static void tearDown() {
+    deleteQuietly(Paths.get(LOGGING_FILE).toFile());
```

Comment:
How is this LOGGING_FILE used?

Reply:
This `LOGGING_FILE` is only being used (as far as I could understand) by the test case [`ApplicationTestCase#applicationWithCustomLogger`](https://github.com/mulesoft/mule-distributions/blob/mule-4.x/embedded/embedded-test/src/test/java/org/mule/runtime/module/embedded/ApplicationTestCase.java#L184)

The reason why I'm kind of "deferring" the "clean up" if due to the error we're getting in Windows: 
```
java.nio.file.FileSystemException: app.log: The process cannot access the file because it is being used by another process.
```
which makes me think, this could be a "reasoble" way of fixing it.